source/reference/configuration.txt

```diff
@@ -12,16 +12,16 @@ Configuration
    :depth: 2
    :class: singlecol
 
-You can configure :program:`mongosync` instances at startup using a
-configuration file.  The configuration file contains settings that are
+You can configure :ref:`mongosync <about-mongosync>` instances at startup 
+using a configuration file.  The configuration file contains settings that are
 the equivalent of ``mongosync`` command line options.
 
 Configuration File
 ==================
 
-Most :program:`mongosync` command line options can be written to a YAML
-file. The configuration file specifies values for each setting using
-YAML format. 
+Most :ref:`mongosync <about-mongosync>` command line options can be 
```

Comment:
[suggestion/question] instead of linking again to the About mongosync page, what do you think about linking "command line options" to the Command Line Options page?

```suggestion
Most :ref:`mongosync <about-mongosync>` command line options can be 
```

Reply:
I like this idea!